test/parallel/test-fs-promises-file-handle-readFile.js

```diff
@@ -35,6 +36,29 @@ async function validateReadFile() {
   await fileHandle.close();
 }
 
+async function validateLargeFileSupport() {
+  const LARGE_FILE_SIZE = 2147483647 + 10 * 1024 * 1024; // INT32_MAX + 10 MB
+  const FILE_PATH = path.join(os.tmpdir(), 'large-virtual-file.bin');
+
+  function createVirtualLargeFile() {
+    return Buffer.alloc(LARGE_FILE_SIZE, 'A');
+  }
```

Comment:
This is quite a large allocation that may fail on some our more resource challenge builders. I believe there's a utility in `test/common` for skipping if there's not enough available memory? Can't recall exactly what it is but we likely should be safer here.

Reply:
thanks I find this method ```enoughTestMem```